internal/maven/artifact.go

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright 2022.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package maven
+
+import (
+	"crypto/sha256"
+	hex2 "encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type Artifact struct {
+	root     string
+	fileName string
+}
+
+// this is useless since md5 broken?
+func (a *Artifact) GetMd5() (string, error) {
```

Comment:
```suggestion
func (a *Artifact) MD5() (string, error) {
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Generally, getters don't have `Get` prefixed either: https://golang.org/doc/effective_go#Getters